store/etcd/rulesets.go

```diff
@@ -36,6 +35,52 @@ func computeLimit(l int) int {
 	return l
 }
 
+// Get returns the ruleset related to the given path. By default, it returns the latest one.
+// It returns the related ruleset version if it's specified.
+func (s *RulesetService) Get(ctx context.Context, path, version string) (*store.RulesetEntry, error) {
+	var (
+		entry *store.RulesetEntry
+		err   error
+	)
+
+	if version == "" {
```

Comment:
This if block could be pushed out to a function with a meaningful name, that could be tested in isolation.   A small change but it would shorten the function an increase legibility.

Reply:
It's a good idea, but it's quite complicated to test it if it's a private method (the new one). 
Moreover, `Latest` and `OneByVersion` methods are not part of the `RulesetService` interface, thus we can't use the `mock` to assert if one or other of these methods have been called (thanks to the `*Count` fields).
If you have a suggestion, just shoot please. 🙏

Reply:
> why are those methods public at all? They're (currently) only used inside this file and in its tests.

They are also used in the `server` package, [here](https://github.com/heetch/regula/blob/issue-24/api/server/api.go#L65) for example, this is the main usage of those methods.
This is why there is [interface](https://github.com/heetch/regula/blob/issue-24/store/service.go#L20) that exposes methods than can be used outside the package (from the `server` one for example) but if you look deeply there is some methods that are not  in it like the `Latest` for example. Once we will need it outside the package we will just have to add it in there.

Reply:
:+1: